src/wwwroot/controllers/TemplateCtrl.js

```diff
@@ -25,22 +26,60 @@
 
     $scope.loadInProgress = false;
     $scope.saveInProgress = false;
+    $scope.isLegacyTemplate = true;
 
-    $scope.saveHtmlRaw = saveHtmlRaw;
+    $scope.save = save;
     $scope.saveAndEditWithMseditor = saveAndEditWithMseditor;
 
     if ($routeParams["templateId"]) {
       load($routeParams["templateId"]);
     }
+    else {
```

Comment:
(very minor) What do you think about moving the _else_ to the same line than the previous curly brace?

```
} else {
```

Reply:
Yes of course, thx